inception/inception-api-annotation-experimental/src/main/java/de/tudarmstadt/ukp/inception/experimental/api/model/Relation.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Licensed to the Technische Universität Darmstadt under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The Technische Universität Darmstadt
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.tudarmstadt.ukp.inception.experimental.api.model;
+
+import java.util.List;
+
+import de.tudarmstadt.ukp.clarin.webanno.api.annotation.model.VID;
+
+public class Relation
+{
+    private VID id;
+    private VID governorId;
+    private VID dependentId;
+    private String governorCoveredText;
```

Comment:
Why send the text along? Shouldn't the UI already have the text and shouldn't it be ably to easily obtain the text via the offsets? I'd rather send along the offsets of the source/target.